<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WaitModalDialog</class>
 <widget class="QDialog" name="WaitModalDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>260</width>
    <height>97</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>FantasyBit Blockchain</string>
  </property>
  <property name="toolTip">
   <string>Â© Satoshi Fantasy, LLC &lt;br&gt;  info@satoshifantasy.com &lt;br&gt; @TF_Ticker &lt;br&gt; @SatoshiFantasy</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" rowspan="2">
    <widget class="QLabel" name="myAnimationLabel">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="myMessageLabel">
     <property name="text">
      <string/>
     </property>
     <property name="textFormat">
      <enum>Qt::PlainText</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="indent">
      <number>2</number>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Block</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="num">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>of</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="height">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QProgressBar" name="progressBar">
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
  </layout>
  <zorder>myMessageLabel</zorder>
  <zorder>progressBar</zorder>
  <zorder>myAnimationLabel</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
